interfaces.go

```diff
@@ -46,7 +47,7 @@ type Breadcrumb struct {
 	Data      map[string]interface{} `json:"data,omitempty"`
 	Level     Level                  `json:"level,omitempty"`
 	Message   string                 `json:"message,omitempty"`
-	Timestamp int64                  `json:"timestamp,omitempty"`
+	Timestamp time.Time              `json:"timestamp,omitempty"`
```

Comment:
We could do this without breaking the API by adding a new field and keeping the old. Since we're still `v0.x` I'd avoid leaving cruft in the exported API.